Rakefile

```diff
@@ -97,6 +100,12 @@ task :setup_env do
   ENV['TF_VAR_client_secret']   = credentials.client_secret
 end
 
+task :check_env do
+  REQUIRED_ENVS.each do |var|
+    abort("Missing ENV: #{var}") unless ENV.key?(var)
+  end
```

Comment:
Probably a better experience for users to see all of the missing vars at once.
```ruby
missing = REQUIRED_ENVS.reject { |var| ENV.key?(var) }

abort("ENV missing: #{missing.join(', ')}") if missing.any?
```

Reply:
👍 Ah good idea!